Task05/Task05/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task05
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            double x = Sinuss(2,3) + Sinuss(5,7) + Sinuss(11,13);
+            Console.WriteLine("x = " + x);
```

Comment:
Округление нужно производить в момент вывода
```suggestion
            Console.WriteLine("x = " + Math.Round(x, 3));
```
А ещё лучше использовать форматирование
```suggestion
            Console.WriteLine($"x = {x:F3}");
```